### DISCUSSION TITLE
Retrieve the color assigned to a layer.

### DISCUSSION THREAD

**Discussion by prestsauce** [DomCR/ACadSharp#186] [Q&A] [answered]
Apologies if this is a stupid question, but I don't quite understand the Color property on the Layer objects. They don't seem to be assigned to anything other than 0 for the documents I'm working with. 

To be clear, I'm looking for the color assignment of the layer as it's shown in the AutoCAD UI. 
![image](https://github.com/DomCR/ACadSharp/assets/20766173/507bee55-088a-4e06-b2e3-04e8945076b4)

Thanks in advance for any insight you can provide.

**Comment by DomCR** [comment, marked as answer]
Hi @prestsauce,

The Color property is not fully implemented, the current progress for this feature is this PR https://github.com/DomCR/ACadSharp/pull/187.

Once is implemented you should be able to get the color form the Layers.

**Reply by prestsauce** [reply]
Thanks for the quick reply.
